packages/design-system-web/.storybook/preview-head.html

```diff
@@ -50,191 +62,191 @@
+                  'type': 'literal',
+                  'xml:lang': 'en',
+                  'value': 'has set up',
+                },
+                count: {
+                  type: 'literal',
+                  datatype: 'http://www.w3.org/2001/XMLSchema#integer',
+                  value: '1',
+                },
               },
               {
-                "predicate": {
-                  "type": "uri",
-                  "value": "https://ontome.net/ontology/p1336"
-                },
-                "predicateLabel": {
-                  "type": "literal",
-                  "xml:lang": "en",
-                  "value": "had arrival place"
-                },
-                "count": {
-                  "type": "literal",
-                  "datatype": "http://www.w3.org/2001/XMLSchema#integer",
-                  "value": "1"
-                }
+                predicate: {
+                  type: 'uri',
+                  value: 'https://ontome.net/ontology/p1336',
+                },
+                predicateLabel: {
+                  'type': 'literal',
+                  'xml:lang': 'en',
+                  'value': 'had arrival place',
+                },
+                count: {
+                  type: 'literal',
+                  datatype: 'http://www.w3.org/2001/XMLSchema#integer',
+                  value: '1',
+                },
               },
               {
-                "predicate": {
-                  "type": "uri",
-                  "value": "https://ontome.net/ontology/p4"
-                },
-                "predicateLabel": {
-                  "type": "literal",
-                  "xml:lang": "en",
-                  "value": "has time-span"
-                },
-                "count": {
-                  "type": "literal",
-                  "datatype": "http://www.w3.org/2001/XMLSchema#integer",
-                  "value": "1"
-                }
-              }
+                predicate: {
+                  type: 'uri',
+                  value: 'https://ontome.net/ontology/p4',
+                },
+                predicateLabel: {
+                  'type': 'literal',
+                  'xml:lang': 'en',
+                  'value': 'has time-span',
+                },
+                count: {
+                  type: 'literal',
+                  datatype: 'http://www.w3.org/2001/XMLSchema#integer',
+                  value: '1',
+                },
+              },
             ],
-            "incomingPropsWithCount": []
+            incomingPropsWithCount: [],
           },
-          "data-entity-props-by-predicate-1": {
-            "loading": false,
-            "entities": [
+          'data-entity-props-by-predicate-1': {
+            loading: false,
+            entities: [
               {
-                "entity": {
-                  "value": "Aabenraa DK",
-                  "type": "literal"
+                entity: {
+                  value: 'Aabenraa DK',
```

Comment:
I suggest you add a longer string here, 
```suggestion
                  value: 'This is a long string, This is a long string',
```

so that you can develop with the geov_entity_props_by_predicate story instead of geov_entity story:

http://localhost:6006/?path=/story/data-components-entity-entity-props-by-predicate--entity-props-with-xsd-string-preloaded